web/styles.css

```diff
@@ -1,5 +1,6 @@
 :root {
     color-scheme: dark;
+    scrollbar-gutter: stable;
```

Comment:
Since 99% of pages will have a scrollbar, this sets that as the expected spacing, so that during loading when the page doesn't have a scrollbar things won't resize randomly, which makes it feel a lot cleaner